### DISCUSSION TITLE
Whiteboard/Handwriting

### DISCUSSION THREAD

**Discussion by darthcircuit** [outline/outline#4963] [Ideas]
I've been using my iPad to take notes for meetings alot, but I would prefer to have all my notes in one place. How feasible would it be to add hand written note capabilities to Outline?

**Comment by Limezy** [comment]
At one point I had developped a tldraw integration within Outline.
Unfortunately, it was removed as tldraw changed their policy and forbid the use of iframes.
However, the new tldraw at beta.tldraw.com allows iframes again, maybe a direction to loo at ?
A simple PR could bring tldraw back to the game, I don't know whether @tommoor would accept it or not

**Comment by Limezy** [comment]
@darthcircuit FYI, #4968 Tldraw beta added in this PR

**Comment by darthcircuit** [comment]
That's fantastic! thank you for that info
